app/src/main/java/com/github/k409/fitflow/features/stepcounter/CaloriesAndDistanceUtil.kt

```diff
@@ -0,0 +1,42 @@
+package com.github.k409.fitflow.features.stepcounter
+
+import android.util.Log
+import com.github.k409.fitflow.di.services.HealthConnectService
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZoneId
+import javax.inject.Inject
+
+class CaloriesAndDistanceUtil @Inject constructor(
```

Comment:
I would suggest renaming this class to enhance its clarity and expressiveness. A possible name could be `HealthStatsManager` or something similar.